doc/source/whatsnew/v0.18.0.txt

```diff
@@ -136,6 +136,89 @@ New Behavior:
    s.index
    s.index.nbytes
 
+.. _whatsnew_0180.enhancements.extract:
+
+Changes to str.extract
+^^^^^^^^^^^^^^^^^^^^^^
+
+The :ref:`.str.extract <text.extract>` method takes a regular
+expression with capture groups, finds the first match in each subject
+string, and returns the contents of the capture groups.
```

Comment:
put this issue number (e.g. the PR number) at the end of this first sentence (as an issue number)